chatmessage/test_chatmessage.py

```diff
@@ -0,0 +1,49 @@
+from .models import Chatmessage
+from account.models.professional import Professional
+from account.models.client import Client
+import pytest
+
+
+@pytest.mark.django_db
+def test_chat_response_code(client):
+    data = {'contact_id': 1, 'sender_id': 3, 'sender_type': 'P'}
+    response = client.get('/chatmessage/', data=data)
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_send_message(client):
+    professional1 = Professional.objects.all()[0]
+    client1 = Client.objects.all()[0]
+
+    contact_id = client1.client_id
+    sender_id = professional1.professional_id
+    sender_type = 'P'
+
+    msg = {'msg_sent': 'Test message!'}
+
+    client.post(f'/chatmessage/?contact_id={contact_id}&sender_id={sender_id}&sender_type={sender_type}', msg)
+
+    last_message_in_chat = Chatmessage.get_chat_between_professional_and_client(professional_id=professional1,
+                                                                                client_id=client1)[0]
+    assert last_message_in_chat.message == 'Test message!'
+
+
+@pytest.mark.django_db
+def test_cant_send_empty_message(client):
+    professional1 = Professional.objects.all()[0]
+    client1 = Client.objects.all()[0]
+
+    contact_id = client1.client_id
+    sender_id = professional1.professional_id
+    sender_type = 'P'
+
+    msg = {'msg_sent': 'The message sent before the empty message'}
+    client.post(f'/chatmessage/?contact_id={contact_id}&sender_id={sender_id}&sender_type={sender_type}', msg)
+
+    empty_msg = {'msg_sent': ''}
+    client.post(f'/chatmessage/?contact_id={contact_id}&sender_id={sender_id}&sender_type={sender_type}', empty_msg)
+
+    last_message_in_chat = Chatmessage.get_chat_between_professional_and_client(professional_id=professional1,
```

Comment:
I think you should tests hear the response behavior when message is empty, 
Looking at views  https://github.com/redhat-beyond/Pro4U/pull/90/files#diff-80cf355c9b63bc64bb2af783117d299cc0e658bd8d52bc57208fcf6d3085d1d6R27

there is no else you render a page without the message sent,

Reply:
The render is outside the if(message!='')
It is done anyway
The condition is intended to know whether to save the new message in the database

Reply:
Yes, you are correct
What happens once the message is empty?
Do you want to add an indication to user, once the message is empty?

Reply:
@Yarboa the views, html and css files have been updated so that when an empty message is sent, a notification will be displayed to the user and also the message will not be sent via chat.